workflows/requests.go

```diff
@@ -6,8 +6,15 @@ type WorkflowReference struct {
 
 // StartRequest is the object describing a Start Workflow request.
 type StartRequest struct {
-	Options           map[string]string `json:"workflow_options"`
-	WorkflowReference WorkflowReference `json:"workflow_reference"`
-	WorkflowName      string            `json:"function_name"`
-	Input             interface{}       `json:"input"`
+	Options      map[string]string `json:"workflow_options"`
```

Comment:
Can we remove WorkflowReference then?

Reply:
Please, follow up on this one and if it can be removed, cut a new PR.

Reply:
Here's the issue to track this suggestion: https://github.com/dapr/dapr/issues/5911